README.md

```diff
@@ -265,7 +265,7 @@ itself) to look for plugins.
 | SQLite                        | [cLupus/asdf-sqlite](https://github.com/cLupus/asdf-sqlite)                                           | [![Build Status](https://travis-ci.org/cLupus/asdf-sqlite.svg?branch=master)](https://travis-ci.org/cLupus/asdf-sqlite)                                                                                          |
 | Stack                         | [sestrella/asdf-stack](https://github.com/sestrella/asdf-stack)                                       | [![CI](https://github.com/sestrella/asdf-stack/workflows/CI/badge.svg?branch=master)](https://github.com/sestrella/asdf-stack/actions)                                                                           |
 | starship                      | [grimoh/asdf-starship](https://github.com/grimoh/asdf-starship)                                       | [![GitHub Actions Status](https://github.com/grimoh/asdf-starship/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/grimoh/asdf-starship/actions)                                           |
-| Steel Bank Common Lisp (sbcl) | [smashedtoatoms/asdf-sbcl](https://github.com/smashedtoatoms/asdf-sbcl)                               | [![Build Status](https://travis-ci.org/smashedtoatoms/asdf-sbcl.svg?branch=master)](https://travis-ci.org/smashedtoatoms/asdf-sbcl)                                                                              |
+| Steel Bank Common Lisp (sbcl) | [smashedtoatoms/asdf-sbcl](https://github.com/smashedtoatoms/asdf-sbcl)                               | [![CI](https://github.com/smashedtoatoms/asdf-sbcl/workflows/CI/badge.svg?branch=master)](https://github.com/smashedtoatoms/asdf-sbcl/actions)                                                        |
```

Comment:
Isn't the URL `Build`, not `CI`?

Reply:
I went off of the ones immediately around mine, which don't appear to be standardized.  I will set it to `Build`.  Sorry about that.

Reply:
No need to apologize! By the way, it depends on the workflow name.

Reply:
I think I got it sorted, but it looks like everything is a little different.  I am not sure there is a standard anymore.